specs/solid-indexing/index.html

```diff
@@ -32,43 +31,13 @@
-      documents globally so even if we saved time we still have to look at the content of the documents by ourselves to see 
-      if the thing we are searching for is there.
-    </p>
+    <p>Hovewer, when a lot of data is available, gathering information and answering complex questions becomes very ineffiscient without indexes. Indexes are made of meta-data that allow to find data easier and faster. Indexing is a widely used mechanism in all kind of software, especially in databases.</p>
 
-    <p>There is an other Web, the semantic Web, sometimes called the Web of data or the Web 3.0. It works the same as the 
-      traditionnal Web except that information is described to machines so they can "understand" what documents and the 
-      things they contain are about. This way we can directly get the information we need without having to look at the 
-      content of documents. This is made possible thanks to the Resource Description Framework (RDF). On this improved Web, 
-      knowledge can be deduced automatically by machines. For instance, if a document is about a person and the machines 
-      know that a person is a human, they can deduce that this person is a human. Search engines can benefit a lot from the web 
-      of data and some engines already take advantage of it to better respond to our queries.
-    </p>
+    <p>This document proposes an RDF vocabulary to describe these indexes, which are also intended to be published on the Web, to facilitate the search of data by Semantic Web agents. In addition, this document also presents some indexing and index-querying techniques.</p>
 
-    <p>
-      In this document we introduce a reusable RDF vocabulary and techniques to index semantic data. A reusable vocabulary 
-      is great to facilitate interoperability and we hope that you will find it suited to your needs. Even if there is no 
-      search field in your application, search operation are often trigerred for instance when some data is displayed. 
-      Indexing you data should improve the performances of your application. Indexing is also useful to make your data 
-      discoverable.
-    </p>
-
-    <ul>
-      <li>indexer des données privées ?</li>
-      <li>Mettre les pattern à une lettre dans un index. Exemple straight + section optimisation.</li>
-      <li>Parler des différents types d'indexation, single, multiple, fédéré, distribué...</li>
-    </ul>
+    <p>This need for a standard indexing vocabulary appeared in the context of the Solid protocol where several applications interact in the same way with the same data to achieve interoperability. Indeed, as the data is decoupled from the applications, they must agree on a client-to-client protocol to work together. If these applications want to be able to use the same indexes, these must be standardized.</p>
```

Comment:
the first "they" was ambiguous
```suggestion
    <p>This need for a standard indexing vocabulary appeared in the context of the Solid protocol where several applications interact in the same way with the same data to achieve interoperability. Indeed, as the data is decoupled from the applications, these applications must agree on a client-to-client protocol to work together. If they want to be able to use the same indexes, these must be standardized.</p>
```